src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -615,6 +619,27 @@ private Rect ApplyMarginAndAlignments(View view, Rect frame)
 			);
 		}
 
+		protected virtual void AdjustAlignment(View view, ref HorizontalAlignment childHorizontalAlignment,
```

Comment:
maybe `AdjustAlignmentForImage`

Reply:
Yes and no.  It's only used by Images, but it's virtual so it could be overriden in various layouters.

Reply:
😬